include/zephyr/drivers/emul_fuel_gauge.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Backend APIs for the fuel gauge emulators.
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_EMUL_FUEL_GAUGE_H_
+#define ZEPHYR_INCLUDE_DRIVERS_EMUL_FUEL_GAUGE_H_
+
+#include <stdint.h>
+#include <zephyr/drivers/emul.h>
+#include <zephyr/drivers/fuel_gauge.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Fuel gauge backend emulator APIs
+ * @defgroup fuel_gauge_emulator_backend fuel gauge backed emulator APIs
```

Comment:
Add this doxygen group to the fuel_gauge.rst doc.

Reply:
Ahh! Thanks!